opensrp-giz-malawi/build.gradle

```diff
@@ -404,6 +404,8 @@ dependencies {
 
     implementation "androidx.multidex:multidex:2.0.1"
     testImplementation "androidx.work:work-testing:2.7.1"
+    androidTestImplementation 'androidx.test:runner:1.3.0'
```

Comment:
These two are also applied for instrumentation tests that are not changed in this PR. Confirm that we need these and probably change this to `testImplementation`